puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/tfm-ror51-release.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- job:
+    name: tfm-ror51-release
+    project-type: workflow
+    logrotate:
```

Comment:
you configure `buildDiscarder` in he groovy file, so I'd think this is not needed here anymore?